llvm_jni/modules/nback/build.bal

```diff
@@ -0,0 +1,1384 @@
+// Copy of build.bal from compiler with imports changed to jni.llvm. Not used in final build
```

Comment:
Why is this checked in?

Reply:
We need this file (file that import `jni.llvm` instead of `print.llvm` and has the correct interface for `main.bal`) so vs code won't errors.

Reply:
This should be generated by transforming the original build.bal at build time.

Reply:
We do that when building with gradle but but if I symlink this file (similar to other common files) since this import `print.llvm` in the original file vs code show this file as an error

Reply:
Fixed by changing file to a symlink